### ISSUE TITLE
Content tile yellow modifier - Colour contrast issue

### ISSUE THREAD

**Issue by houbly** [wmcadigital/wmn-design-system#274]
When the  yellow modifier is active on the content tile its read more link fails colour contrast accessibility [see below].

<img width="970" alt="Screenshot 2020-08-17 at 10 41 32" src="https://user-images.githubusercontent.com/6360479/90382339-8926b700-e076-11ea-9b3d-b355d97f5f8d.png">

<img width="753" alt="Screenshot 2020-08-17 at 10 41 19" src="https://user-images.githubusercontent.com/6360479/90382330-862bc680-e076-11ea-85c9-51d920ce3b46.png">
